android/logto-client/src/main/kotlin/io/logto/sdk/android/LogtoClient.kt

```diff
@@ -21,6 +25,94 @@ open class LogtoClient(
 
     fun isAuthenticated() = idToken != null
 
+    fun getAccessToken(callback: RetrieveCallback<AccessToken>) =
+        getAccessToken(null, null, callback)
+
+    fun getAccessToken(
```

Comment:
will it be better to add some comments for a more clear demonstration? since this function is large. e.g.

```kt
// Parameter validation
// ...
// Fetch access token by refresh token
```

Reply:
updated